astropy/io/ascii/cds.py

```diff
@@ -21,8 +22,56 @@
 from astropy.units import Unit
 
 
+from astropy.io import ascii
+from astropy.table import Table
+from .CDSColumn import CDSColumn
+import numpy as np
+import sys
+from string import Template
+from textwrap import wrap, fill
+import datetime
+import logging
+import math
+
+MAX_SIZE_README_LINE = 80
+MAX_COL_INTLIMIT = 10000000
+
+
 __doctest_skip__ = ['*']
 
+cdsdicts = {'title': 'Title ?',
+            'author': '1st author ?',
+            'catalogue': '',
+            'date': 'Date ?',
+            'abstract': 'Abstract ?',
+            'authors': 'Authors ?',
+            'bibcode': 'ref ?',
+            'keywords': ''
+            }
+
+
+class CdsSplitter(fixedwidth.FixedWidthSplitter):
+    """
+    Contains the join function to left align the CDS columns
+    when writing to a file.
+    """
+    delimiter_pad = ''
+    bookend = False
+    delimiter = ' '
+
+    def join(self, vals, widths):
+        pad = self.delimiter_pad or ''
+        delimiter = self.delimiter or ''
+        padded_delim = pad + delimiter + pad
+        if self.bookend:
+            bookend_left = delimiter + pad
+            bookend_right = pad + delimiter
+        else:
+            bookend_left = ''
+            bookend_right = ''
+        vals = [val + ' ' * (width - len(val)) for val, width in zip(vals, widths)]
```

Comment:
I am wondering if this could just be added with a fixed delimiter/pad and alignment which would simplify this code for writing.